app/models/setting.rb

```diff
@@ -13,4 +13,14 @@ def self.get(key)
     value_object = Setting.where(key: key, status: "Active").first
     value_object.value if value_object
   end
+
+  def active?
+    self.status == 'Active'
+  end
+
+  def self.get_value(key, fallback_value = nil)    
```

Comment:
This would be even better if we went to `get_integer`, as it would show intent even better. Then, as needed, we can add get_boolean , for example. These should always return the datatype or nil.

Reply:
Yeah makes sense, do you also want me to add, get_boolean and get_string as well.

Reply:
If you want, or we can wait til we need them.